crates/rspack_core/src/context_module.rs

```diff
@@ -847,7 +847,7 @@ impl Module for ContextModule {
     &[SourceType::JavaScript]
   }
 
-  fn original_source(&self) -> Option<&dyn rspack_sources::Source> {
+  fn source(&self) -> Option<&rspack_sources::BoxSource> {
```

Comment:
prefer original_source, imply it is the input from user land

Reply:
The reason why original_source is merged into source is that source will not be modified after build is assigned, so I think it is OK to use source directly.